smarts/core/road_map.py

```diff
@@ -76,6 +76,20 @@ def dynamic_features(self) -> List[RoadMap.Feature]:
         """All dynamic features associated with this road map."""
         return []
 
+    @staticmethod
+    def serialize(road_map: "RoadMap") -> Any:
+        """The default serialization for the road map."""
+        import cloudpickle
+
+        return cloudpickle.dumps(road_map)
```

Comment:
How did you get around the issues related to lanepoints that we were seeing with this?

Reply:
I ended up using a proxy object to format and then reconstruct the `road_map`.

https://github.com/huawei-noah/SMARTS/blob/edd90dbcc6af5529a3dba0fd83ea93be43b96eaa/smarts/core/serialization/default.py#L28-L80

Reply:
I looked around, it looks like `gymnasium` has a potentially better idea for this: https://github.com/Farama-Foundation/Gymnasium/blob/c2a387702c48d2e50f499a4f47d30e293ad75240/gymnasium/utils/ezpickle.py#L4